src/api.cc

```diff
@@ -173,3 +235,20 @@ bool AkliteClient::IsRollback(const TufTarget& t) const {
 
   return known_local_target(*client_, target, known_but_not_installed_versions);
 }
+
+InstallResult AkliteClient::SetSecondaries(const std::vector<SecondaryEcu>& ecus) {
+  std::vector<std::string> hwids;
+  Json::Value data;
+  for (const auto& ecu : ecus) {
+    Json::Value entry;
+    entry["target"] = ecu.target_name;
+    data[ecu.serial] = entry;
+    hwids.emplace_back(ecu.hwid);
+  }
```

Comment:
Conversions of the  initial/input ecu vector (`std::vector<SecondaryEcu>& ecus` ) seems like redundant. Maybe we could send  it to the backend as it is as well as store it in the context in the same format.

It's not critical, just optimization and unification.